client/galaxy/scripts/mvc/ui/ui-misc.js

```diff
@@ -78,6 +78,57 @@ define(['utils/utils',
         }
     });
 
+
+    	var Password = Backbone.View.extend({
```

Comment:
Ok then yes I think we could reuse it, I will test it and commit it if this works!